Bottom: 7cdee970e9fe138fb20e5981eec4bae45d1bf8b5
Top:    6815f7191adc7c5872ea7865d7af6ed44eb90b5d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 18:48:55 +0000

toporouter.c: Remove un-necessary call to ResetConnections()

This is not required by the call to CollectSubnets() which follows it,
and we don't have any other requirement to clear flags.

Futhermore, as toporouter.c does not setup which flag is being operated
on, the effect of this call will depend on which flag has been left in
"TheFlag" variable in find.c.


---

diff --git a/src/toporouter.c b/src/toporouter.c
index 0a83569..75ce82b 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -2543,7 +2543,6 @@ void
 import_clusters(toporouter_t *r)
 {
   NetListListType nets;
-  ResetConnections (false);
   nets = CollectSubnets(false);
   NETLIST_LOOP(&nets);
   {
